java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStoreIT.java

```diff
@@ -30,8 +30,17 @@
 import sleeper.core.statestore.FileReferenceFactory;
 import sleeper.core.statestore.SplitFileReference;
 import sleeper.core.statestore.SplitFileReferences;
+import sleeper.core.statestore.SplitRequestsFailedException;
 import sleeper.core.statestore.StateStore;
 import sleeper.core.statestore.StateStoreException;
+import sleeper.core.statestore.exception.FileAlreadyExistsException;
+import sleeper.core.statestore.exception.FileHasReferencesException;
+import sleeper.core.statestore.exception.FileNotFoundException;
+import sleeper.core.statestore.exception.FileReferenceAlreadyExistsException;
+import sleeper.core.statestore.exception.FileReferenceAssignedToJobException;
+import sleeper.core.statestore.exception.FileReferenceNotAssignedToJobException;
+import sleeper.core.statestore.exception.FileReferenceNotFoundException;
+import sleeper.core.statestore.exception.NewReferenceSameAsOldReferenceException;
```

Comment:
There are some tests missing from this class that are in the other file reference store test classes. Please can this be aligned so that we can copy and paste all the tests at once with minimal changes?

Reply:
Thanks, i've realigned this test class with the others, the same tests should be in all 3 now

Reply:
There are a bunch of places where the exception thrown is inconsistent.

There are places where it asserts on DynamoDB-specific exceptions, which I'm not sure is necessary.

There are places where the way files are added in tests is inconsistent, ie. addFilesWithReferences is used instead of addFiles or vice versa.